index.js

```diff
@@ -33,27 +33,58 @@ module.exports = function Addon(manifest) {
 	const manifestBuf = new Buffer.from(JSON.stringify(manifest))
 	if (manifestBuf.length > 8192) throw 'manifest size exceeds 8kb, which is incompatible with addonCollection API'
 
-	// Serve the manifest
-	addonHTTP.get('/manifest.json', function (req, res) {
-		res.setHeader('Content-Type', 'application/json; charset=utf-8')
-		res.end(manifestBuf)
-	})
+	function getParams(req) {
+		if (req.params && req.params.resource) {
+			return req.params
+		} else {
```

Comment:
why are you not leveraging the routing functionality of serverless, as I suggested earlier?